utils/docker/Dockerfile.leap.15

```diff
@@ -86,71 +93,60 @@ RUN zypper --non-interactive update && \
-            branch="master";                                                                                     \
-            build_number="lastSuccessfulBuild";                                                                  \
-            if [[ $repo = *@* ]]; then                                                                           \
-                branch="${repo#*@}";                                                                             \
-                repo="${repo%@*}";                                                                               \
-                if [[ $branch = *:* ]]; then                                                                     \
-                    build_number="${branch#*:}";                                                                 \
-                    branch="${branch%:*}";                                                                       \
-                fi;                                                                                              \
-            fi;                                                                                                  \
-            zypper --non-interactive addrepo --no-gpgcheck                                                       \
-                   ${JENKINS_URL}job/daos-stack/job/$repo/job/$branch/$build_number/artifact/artifacts/leap15/   \
-                   $repo:$branch:$build_number;                                                                  \
-        done;                                                                                                    \
-        echo "Installing: $QUICKBUILD_DEPS";                                                                     \
-        echo "$QUICKBUILD_DEPS" | tr '\n' '\0' | xargs -0 zypper --non-interactive install;                      \
+RUN if $QUICKBUILD; then                                                       \
+        for repo in $REPOS; do                                                 \
+            branch="master";                                                   \
+            build_number="lastSuccessfulBuild";                                \
+            if [[ $repo = *@* ]]; then                                         \
+                branch="${repo#*@}";                                           \
+                repo="${repo%@*}";                                             \
+                if [[ $branch = *:* ]]; then                                   \
+                    build_number="${branch#*:}";                               \
+                    branch="${branch%:*}";                                     \
+                fi;                                                            \
+            fi;                                                                \
+            zypper --non-interactive addrepo --no-gpgcheck                     \
```

Comment:
Again, shouldn't be disabling GPG checking at the command level.  It will result in installing unchecked upstream packages.

Reply:
This is for a Jenkins generated repo, which does not have any GPG signatures.